Source/Core/pages/js/browserServices.js

```diff
@@ -7,104 +7,116 @@ browserServices.checkBrowser = function() {
 		browser = 'chrome';
 	if ('firefoxApi' in window)
 		browser = 'firefox';
-	if ('safariApi' in window)
-		browser = 'safari';
-
+  /**
+   * detecting SAFARI browser
+   * 'chrom' filters down (chrom)e as well as (chrom)ium 
+   */
+  if(!(navigator.userAgent.toLowerCase().indexOf("chrom") != -1)){
```

Comment:
Firefox uses a different background page than this so it is decided earlyer down the Road